package.json

```diff
@@ -10,8 +10,8 @@
   "scripts": {
     "build": "lerna run build",
     "build:production": "lerna run build:production",
-    "download:engine": "ts-node scripts/download-engine.ts",
-    "download:openapi": "ts-node scripts/download-openapi.ts",
+    "download:engine": "node --experimental-strip-types scripts/download-engine.cts",
+    "download:openapi": "node --experimental-strip-types scripts/download-openapi.ts",
```

Comment:
node has meanwhile builtin type striping, even if experimental for this it should to it's job :)

Reply:
aha cool